docs/content/python/getting-started/quickstart/data/data.md

```diff
@@ -42,24 +41,24 @@ objects: Python objects
 methods: Python methods
 register_classes: |
     ```python
+    class Page:
```

Comment:
I like the simplicity of this, but it's not too realistic. What if we parameterize them?

```
class Page:
    def __init__(self, contents):
        self.contents = contents

class User:
    def __init__(self, role):
        self.role = role
```

Then, we could have a few examples of calling the policy:

```
assert oso.is_allowed(User(role="guest"), "read", Page("readable page"))
assert not oso.is_allowed(User(role="guest", "write", Page("readable page"))
assert oso.is_allowed(User(role="admin", "write", Page("readable page"))
```

We may not need any content for the page, we could just have it be an empty class. What do you think of that?

Reply:
That's a great suggestion — I was uncomfortable with how "stub-implementation" those felt.